finding/finding.go

```diff
@@ -181,6 +182,24 @@ func (f *Finding) WithMessage(text string) *Finding {
 	return f
 }
 
+// UniqueProbesEqual checks the probe nams present in a list of findings
+// and compare them against an expected list.
+func UniqueProbesEqual(findings []Finding, probes []string) bool {
```

Comment:
this PR is branched off of #3241? I'll wait for that to be resolved so I don't duplicate

Reply:
sg.